__tests__/helpers.ts

```diff
@@ -5,6 +5,7 @@ import type { RequestOptions, MockResponse } from 'node-mocks-http'
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export const cleanDatabase = async (_prisma: PrismaClient) => {
   // NOTE: Put here your cleanup instructions
+  await _prisma.company.deleteMany()
```

Comment:
You can replace this function with the following example:

```
export const cleanDatabase = async (prisma: PrismaClient) => {
  const deleteTransactions = prisma.transaction.deleteMany()
  const deleteEmployees = prisma.employee.deleteMany()
  const deleteCompanies = prisma.company.deleteMany()

  await prisma.$transaction([
    deleteTransactions,
    deleteEmployees,
    deleteCompanies,
  ])
}
```

Please note that we don't need `eslint-disable-next-line` any more.

Reply:
My bad, there is no `transaction` table, but `payment`. I'll fix it myself.